embulk-standards/src/main/java/org/embulk/standards/StdoutOutputPlugin.java

```diff
@@ -40,6 +45,15 @@ public ConfigDiff transaction(ConfigSource config,
             Schema schema, int taskCount,
             OutputPlugin.Control control) {
         final PluginTask task = config.loadConfig(PluginTask.class);
+        if (task.getPrintsColumnNames()) {
+            for (final Column column : schema.getColumns()) {
+                if (column.getIndex() > 0) {
+                    System.out.print(",");
+                }
+                System.out.print(column.getName());
+            }
+            System.out.println("");
```

Comment:
Why does this empty output is needed?
Is it intended to ":" ?

Reply:
Ah, no. It's newline. One option was to use `\n`, but `println` might be better.

Reply:
Ah Ok. I overlooked print"ln".